client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/FlightEventForm.tsx

```diff
@@ -83,11 +88,16 @@ const FlightEventForm : React.FC = () : JSX.Element => {
                     <FormInput fieldName='ארץ יעד'>
                     <Autocomplete
                         options={Array.from(countries, ([id, value]) => ({ id, value }))}
-                        getOptionLabel={(option) => option.value.displayName}
-                        inputValue={countries.get(flightDestinationCountry as string)?.displayName}
```

Comment:
why removing this?

Reply:
I replaced it with default value, because doing it that way causes it not to change on backspacing and typing